components/raftstore/src/store/worker/region.rs

```diff
@@ -675,6 +675,14 @@ where
         }
     }
 
+    /// Called when received new ApplyTask
+    fn handle_new_pending_applies(&mut self) {
```

Comment:
Please give a comment to explain when should call 'handle_new_pending_applies' and when should call 'handle_pending_applies'.

Reply:
This function is kind of ambiguous, I am trying to remove that.